docs/content/en/hosting-and-deployment/hosting-on-tiiny.md

```diff
@@ -0,0 +1,48 @@
+---
+title: Host on Tiiny.Host
+linktitle: Host on Tiiny.Host
+description: You can use tiiny.host to deploy your static hugo website, no registration required.
+date: 2020-05-19
+publishdate: 2020-05-19
+lastmod: 2017-03-15
+categories: [hosting and deployment]
+keywords: [hosting, tiiny]
+authors: [Elston Baretto]
+menu:
+  docs:
+    parent: "hosting-and-deployment"
+    weight: 20
+weight: 20
+sections_weight: 20
+draft: false
+toc: true
+aliases: []
+---
+
+## Assumptions
+
+You have completed the [Quick Start][] or have a completed Hugo website ready for deployment.
```

Comment:
You're missing a link for this.

Reply:
Thanks for spotting that. I used the firebase template as a guide and just noticed that the Quick Start link is broken there too: https://github.com/gohugoio/hugo/blob/master/docs/content/en/hosting-and-deployment/hosting-on-firebase.md

Would you like me to update that too?

Reply:
In fact its broken in many places, here too: `[Quick Start][]` https://github.com/gohugoio/hugo/edit/master/docs/content/en/hosting-and-deployment/hosting-on-github.md

Reply:
I've updated it using references as per the other docs

Reply:
Hey @moorereason is there anything else you'd like me to add?

Thanks for the review!